chaste_codegen/templates/Cvode/cpp/constructor

```diff
@@ -1,6 +1,6 @@
    
     {{class_name}}::{{class_name}}(boost::shared_ptr<AbstractIvpOdeSolver> pOdeSolver /* unused; should be empty */, boost::shared_ptr<AbstractStimulusFunction> pIntracellularStimulus)
-        : {{base_class}}(
+        : {% if modifiers|length > 0 %}AbstractCardiacCellWithModifiers<{% endif%}{{base_class}}{%- if modifiers|length > 0 %} >{% endif%}(
```

Comment:
Wouldn't it be better for the Python code/parent template to pass the correct base_class in?

Reply:
Also, could a CVODE cell work with this base class?

Reply:
OK, CVODE should work (see comments below) but some won't.

Reply:
I was going to argue against it but actually the more I think about it the more I realise this is a great idea, as it allows bringing some more templates together